addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -88,4 +96,43 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
     this.hasBeenFocused = true;
     super.updateValidations();
   }
+
+  getFieldOptionsAsObject(fieldOptions) {
+    if (typeof fieldOptions == 'string') {
+      try {
+        return JSON.parse(fieldOptions.trim());
+      } catch (error) {
+        console.error(` | Could not parse string options to json object`);
+        console.error(` | `, error);
+
+        return {};
+      }
+    }
+
+    return fieldOptions;
+  }
+
+  isValidFieldOptions(fieldOptions) {
```

Comment:
nitpick

```suggestion
  hasValidFieldOptions(fieldOptions) {
```

nitpick 2, I prefer to use functions instead of methods if all the required data is passed in. It feels easier to follow, but maybe that's just personal / what I'm used to.